<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QT_SemesClass</class>
 <widget class="QMainWindow" name="QT_SemesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QT_Semes</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPridat">
    <property name="title">
     <string>Pridať</string>
    </property>
    <addaction name="actionOsoba_2"/>
    <addaction name="actionTest_3"/>
    <addaction name="separator"/>
    <addaction name="actionGenerator"/>
   </widget>
   <widget class="QMenu" name="menuZobrazi">
    <property name="title">
     <string>Zobraziť</string>
    </property>
    <addaction name="actionOsoba"/>
    <addaction name="actionTest"/>
    <addaction name="actionLokality"/>
   </widget>
   <widget class="QMenu" name="menuTester">
    <property name="title">
     <string>Tester</string>
    </property>
    <addaction name="actionTest_2"/>
   </widget>
   <widget class="QMenu" name="menuS_bor">
    <property name="title">
     <string>Súbor</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
   </widget>
   <addaction name="menuS_bor"/>
   <addaction name="menuPridat"/>
   <addaction name="menuZobrazi"/>
   <addaction name="menuTester"/>
  </widget>
  <action name="actionGenerator">
   <property name="text">
    <string>Generátor</string>
   </property>
  </action>
  <action name="actionManu_lne">
   <property name="text">
    <string>Manuálne</string>
   </property>
  </action>
  <action name="actionGenerova">
   <property name="text">
    <string>Generovať</string>
   </property>
  </action>
  <action name="actionSlovensko">
   <property name="text">
    <string>Slovensko</string>
   </property>
  </action>
  <action name="actionKraj">
   <property name="text">
    <string>Kraj</string>
   </property>
  </action>
  <action name="actionOkres">
   <property name="text">
    <string>Okres</string>
   </property>
  </action>
  <action name="actionOsoba">
   <property name="text">
    <string>Osoba</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionLokality">
   <property name="text">
    <string>Lokality</string>
   </property>
  </action>
  <action name="actionSlovensko_2">
   <property name="text">
    <string>Slovensko</string>
   </property>
  </action>
  <action name="actionSlovensko_3">
   <property name="text">
    <string>Slovensko</string>
   </property>
  </action>
  <action name="actionKraje">
   <property name="text">
    <string>Kraje</string>
   </property>
  </action>
  <action name="actionOkresy">
   <property name="text">
    <string>Okresy</string>
   </property>
  </action>
  <action name="actionTest_2">
   <property name="text">
    <string>Test štruktúry</string>
   </property>
  </action>
  <action name="actionOsoba_2">
   <property name="text">
    <string>Osoba</string>
   </property>
  </action>
  <action name="actionTest_3">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QT_Semes.qrc"/>
 </resources>
 <connections/>
</ui>
